.gitignore

```diff
@@ -0,0 +1,68 @@
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+*.ipch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
```

Comment:
We don't have any fortran sources

Reply:
yep.  this is a stock file covering common issues across different types of projects.